File: src/clojure/leiningen/new/mini2dx_java/DesktopLauncher.java
package {{package-name}};

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import org.mini2Dx.core.game.Mini2DxGame;

public class DesktopLauncher {
	public static void main (String[] args) {
		LwjglApplicationConfiguration cfg =
			new LwjglApplicationConfiguration();
		cfg.title = "{{class-name}}";
		cfg.useGL20 = true;
		cfg.width = 800;
		cfg.height = 600;
		//cfg.useCPUSynch = false;
		cfg.vSyncEnabled = true;
		new LwjglApplication(
			new Mini2DxGame(new {{class-name}}()),
			cfg
		);
	}
}

File: src/clojure/leiningen/new/libgdx_clojure/desktop-launcher.clj
(ns {{desktop-namespace}}
  (:require [{{namespace}}])
  (:import [com.badlogic.gdx.backends.lwjgl LwjglApplication])
  (:gen-class))

(defn -main
  []
  (LwjglApplication. ({{namespace}}.Game.) "{{app-name}}" 800 600 true))
